source/modules/MEI.header.xml

```diff
@@ -107,6 +107,29 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.perfRes" module="MEI.header" type="atts">
```

Comment:
Are you all sure about merging bot attributes of `perfRes` and `perfResList` into one attribute class that is being made available on both elements? This means `@adLib` and `@solo` will also be available o `perfResList`. I guess you spent some thought on this but could you please enlighten me?

Reply:
Don't get me wrong, I like the idea to have a single definition of `@count` and reuse it wherever possible ;-)

Reply:
The discussion about this starts [obove](https://github.com/music-encoding/music-encoding/pull/842#discussion_r675477539).
Thanks for your advice. When I review this now I'm not sure if it is right if `<perfResList>` can have `@solo`. This problem arises when we joined the attribute classes I modeled at first.
-> In the las ODD meeting we decided to rename `@adLib`and to move it to an own attribute class. So I'm going to think about this when I do the changes.